extensions/fairifier-rdf-extension/dtls/fairifier/PostFairDataToFairDataPoint.java

```diff
@@ -194,7 +197,9 @@ private DatasetMetadata addPropertiesToDataset(JSONObject dataset, String uuid_d
             datasetMetadata.setContactPoint(FACTORY.createIRI(dataset.getString(CONTACTPOINTPREDICATE)));
         }
         if (dataset.has(LANGUAGEPREDICATE)) {
-            datasetMetadata.setLanguage(FACTORY.createIRI(dataset.getJSONArray(LANGUAGEPREDICATE).getString(0)));
+            if (!dataset.getString(LANGUAGEPREDICATE).equals("[\"\"]")) {
```

Comment:
Same comment as for the catalog languages, maybe there's another way to check this?